### PR TITLE
Transcript edit update

### LINKED ISSUE #19: transcript edit page punchlist
The youtube link should be a _blank href, the button label should be renamed "source", the color should be something like the save button or grey.

The save button doesn't seem to do anything and should be wired up to save the latest state to the content blob

The restore original button should be a grey outlined button and have a confirmation, "are you sure, any changes you've made will be lost"

I should be able to type a date rather than be forced to use the date picker

We don't need the "Transcription Text" div

We want a full-width markdown editor with a preview tab rather than 2 side by side panels

There are a bunch of unnecessary buttons on the markdown editor. Remove insert image, table, mermaid, epsilon, "prettier", html preview, link to github project, etc. It should just be the basics.

### PR DESCRIPTION
**Pull request by Emmanuel-Develops**
closes #19

### CODE REVIEW

**Comment by adamjonas** [comment]
Can't test because I can't get to the edit page locally.

File: frontend-next/src/components/editTranscript/EditTranscript.tsx [hunk 1]
```diff
@@ -37,27 +59,87 @@ const EditTranscript = ({
...
+        <ModalOverlay />
+        <ModalContent>
+          <ModalHeader>
+            <Text align="center" fontWeight={600} fontSize="md" color="red.600">
+              Restore Original
+            </Text>
+          </ModalHeader>
+          <ModalCloseButton />
+          <ModalBody>
+            <Text>Are you sure? any changes made will be lost.</Text>
```
Comment (adamjonas) [inline frontend-next/src/components/editTranscript/EditTranscript.tsx:120]:
```suggestion
            <Text>Are you sure? All changes will be lost.</Text>
```

**Comment by adamjonas** [comment]
The outline button doesn't function. Cut it.

Screen Shot 2023-03-30 at 6 35 41 PM

**Comment by adamjonas** [comment]
Looks like the save button still doesn't work. Can we link that up?

**Comment by adamjonas** [comment]
Cut this button whatever that is.
Screen Shot 2023-03-30 at 6 42 43 PM

**Comment by adamjonas** [comment]
what does this save button do?

Screen Shot 2023-03-30 at 6 43 49 PM

**Comment by Emmanuel-Develops** [comment]
> Looks like the save button still doesn't work. Can we link that up?

I haven't set it up yet. I wanted to run it up with @nassersaazi the distinction between save and submit. Given there's a PUT request to update data. I assume that should be save. Is that right @nassersaazi ?

**Comment by Emmanuel-Develops** [comment]
> what does this save button do?
> 
> Screen Shot 2023-03-30 at 6 43 49 PM

It is tied to the editor and triggers an onSave event which does nothing currently. But I can make both buttons (Save on the sideBar and savebutton) call the same function to save. With the distinction that editor-save, saves only the body (editor text) while the save in sidebar saves all edits including sidebaredit (e.g speakers)

**Comment by Emmanuel-Develops** [comment]
> Cut this button whatever that is. Screen Shot 2023-03-30 at 6 42 43 PM

This is to write equations, which could be useful if a media has a section where they disuss technical details.

**Comment by adamjonas** [comment]
Cut the equations and the save button on the editor. The fewer buttons, the better.

**Comment by adamjonas** [comment]
I'm ok with merging without the save button in place to keep things moving. Would like to cut down on the buttons and update copy.

**Comment by Emmanuel-Develops** [comment]
> What does this thing do? If nothing, let's remove it. Otherwise, let's merge
> 
> Screen Shot 2023-03-31 at 9 06 19 AM

When they were side-by-side it ensures the two pages were synced when one was scrolled. It still helps when we have a tabSwitcher as at the point when you click preview it shouldn't start from the top if there had been a scroll.
You can toggle the option then toggle preview and see how it works. I can enable it be default and remove the option to toggle it

**Pull request merged**